Composer/plugins/samples/assets/shared/scripts/provisionComposer.js

```diff
@@ -170,6 +171,23 @@ const unpackObject = output => {
   return unpacked;
 };
 
+const getTenantId = async (accessToken) => {
+  if (!accessToken) {
+    throw new Error('Invalid Access Token');
+  }
+  try {
+    const tenantUrl = `https://management.azure.com/tenants?api-version=2020-01-01`;
+    const options = {
+      headers: { Authorization: `Bearer ${accessToken}` },
+    };
+    const response = await rp.get(tenantUrl, options);
+    const jsonRes = JSON.parse(response);
+    return jsonRes.value[0].tenantId;
+  } catch (err) {
+    throw new Error(`Get Tenant Id Failed`);
```

Comment:
Same comment as before, this error should not be swallowed, `err` has valuable information for the user and for support engineers helping users troubleshoot. b